Development/src/pages/Dashboard.js

```diff
@@ -10,8 +10,8 @@ import Settings from '../settings';
 export default () => (
     <div>
         <Card>
-            <Typography variant="display3" component="h2" align="center">
-                Welcome to the nmos-js Client
+            <Typography variant="display3" align="center">
+                Welcome to the nmos&#8209;js Client
```

Comment:
:+1: Shame there isn't an HTML entity like `&nbsp;` for non-breaking hyphen!

Reply:
I'll leave a comment on the line above